packages/react-native-sdk/package.json

```diff
@@ -29,43 +29,42 @@
     "test": "jest"
   },
   "dependencies": {
-    "axios": "0.19.0",
-    "kinvey-js-sdk": "^5.1.1",
-    "lodash": "^4.17.15",
-    "pubnub": "4.25.2"
+    "axios": "0.21.1",
+    "kinvey-js-sdk": "6.0.0-next.1",
+    "lodash": "4.17.21",
+    "pubnub": "4.32.1"
   },
   "peerDependencies": {
-    "@react-native-community/async-storage": "^1.6.1",
-    "react-native": "^0.60.5",
-    "react-native-push-notification": "^3.1.9"
+    "@react-native-async-storage/async-storage": "1.15.4",
```

Comment:
It may be better to keep using the caret range for the peer dependency versions. Allowing users to update them without warnings.